javascript/javascript2/week2/hyfBayHelpers.js

```diff
@@ -0,0 +1,72 @@
+/* DONT MODIFY ANY OF THIS CODE!!!*/
+
+window.getAvailableProducts = function () {
+  function getRandomInt(min, max) {
+    min = Math.ceil(min);
+    max = Math.floor(max);
+    return Math.floor(Math.random() * (max - min + 1)) + min;
+  }
+
+  function getRandomItem(availableProductNames) {
+    return availableProductNames[
+      getRandomInt(0, availableProductNames.length - 1)
+    ];
+  }
+
+  function getRandomProductname() {
+    const preWords = [
+      "Used",
+      "Fantastic",
+      '"Used"',
+      "Broken",
+      "Beautiful",
+      "Wet",
+      "Green",
+      "Sloppy",
+      "Dirty",
+    ];
+    const productNames = [
+      "Carrot",
+      "Drone",
+      "Giftcard",
+      "Puppy",
+      "Car",
+      "Shirt",
+      "Milk",
+      "Chalk",
+      "Fish fingers",
+      "Socks",
+      "Chocolate",
+      "Toothbrush",
+      "Computer",
+      "Nokia",
+      "Cologne",
```

Comment:
no need of comma '**,**' in last index of array

Reply:
Finally, I found the setting of Prettify, that disables obligatory trailing comas. Hope you won't see them next time :)